File: AIAP_LAB_18/NumberCheck.java
/*Write a Java method checkNumber(int num) that checks if a number is positive, negative, or zero.*/
public class NumberCheck {
    public static void checkNumber(int num) {
        if (num > 0) {
            System.out.println(num + " is a positive number.");
        } else if (num < 0) {
            System.out.println(num + " is a negative number.");
        } else {
            System.out.println("The number is zero.");
        }
    }

    public static void main(String[] args) {
        checkNumber(5);  // Example usage
        checkNumber(-7);  // Example usage
        checkNumber(0);   // Example usage
    }
}

File: AIAP_LAB_18/NumberCheck.py
#Translate the method present in number.java into a Python function check_number(num).
def check_number(num):
    if num > 0:
        print(f"{num} is a positive number.")
    elif num < 0:
        print(f"{num} is a negative number.")
    else:
        print(f"{num} number is zero.")
# Example usage
check_number(5)
check_number(-7)
check_number(0) 
